packages/prisma/migrations/20230921002822_fix_booking_time_status/migration.sql

```diff
@@ -0,0 +1,35 @@
+-- View: public.BookingsTimeStatus
+
+-- DROP VIEW public."BookingsTimeStatus";
+
+CREATE OR REPLACE VIEW public."BookingTimeStatus"
+ AS
+ SELECT "Booking".id,
+    "Booking".uid,
+    "Booking"."eventTypeId",
+    "Booking".title,
+    "Booking".description,
+    "Booking"."startTime",
+    "Booking"."endTime",
+    "Booking"."createdAt",
+    "Booking".location,
+    "Booking".paid,
+    "Booking".status,
+    "Booking".rescheduled,
+    "Booking"."userId",
+    "et"."teamId",
+	  "et"."length" as "eventLength",
+      CASE
+          WHEN "Booking".rescheduled IS TRUE THEN 'rescheduled'::text
+          WHEN "Booking".status = 'cancelled'::"BookingStatus" AND "Booking".rescheduled IS NULL THEN 'cancelled'::text
```

Comment:
The main change here is we were looking for "Booking".rescheduled IS FALSE which was incorrect

Reply:
Views need to be recreated since cannot be updated

Reply:
Nice SQL-fu in here

![Noice](https://media1.giphy.com/media/yJFeycRK2DB4c/200.gif)